examples/sunpy_other_packages/reprojection_different_observers.py

```diff
@@ -133,13 +134,15 @@
     scale=u.Quantity(map_aia.scale)*4,
     rotation_matrix=map_aia.rotation_matrix,
     instrument="AIA",
+    telescope=map_aia.observatory,
     wavelength=map_aia.wavelength
 )
 
 ######################################################################
 # Once again we need to generate a `~astropy.wcs.WCS` and then manually
 # set the observer location.
 
+print('header:', mars_header)
```

Comment:
Why was this added?

Reply:
I am extremely sorry, I added it while debugging, and forgot to remove

Reply:
No problem 😅 surprised we didn’t catch it before